config/config.js

```diff
@@ -4,8 +4,7 @@ let config
 
 if (process.env.NODE_ENV === 'dev') {
   config = {
-    'database': 'MongoDB',
-    'dbUrl': `mongodb://localhost:27017/`,
+    'dbUrl': `mongodb://localhost:27017/testnew`,
```

Comment:
Possible to change this back or to something less casual for `testnew`

Reply:
changed it to devex-apollo. 
Good point, also if the user has more databases running on his local machine we should avoid conflicts.